project-ideas/fulfillment-center-mgmt/approveTransferOrder.md

```diff
@@ -0,0 +1,67 @@
+# Approve Transfer Orders
```

Comment:
This could be generic approve order service, and should be moved to oms component.